zerver/lib/handlers.py

```diff
@@ -6,9 +6,11 @@
 handlers = {} # type: Dict[int, Any] # TODO: Should be AsyncDjangoHandler but we don't important runtornado.py.
 
 def get_handler_by_id(handler_id):
+    # type: (int) -> Any # TODO: type of vaue of handlers
```

Comment:
this would be an AsyncDjangoHandler from zerver/management/commands/runtornado.py.  We'll want to refactor that to be in its own file before annotating this will make sense.